pkg/services/sqlstore/user.go

```diff
@@ -435,7 +435,8 @@ func SearchUsers(query *models.SearchUsersQuery) error {
 
 	whereConditions := make([]string, 0)
 	whereParams := make([]interface{}, 0)
-	sess := x.Table("user")
+	sess := x.Table("user").Alias("u")
+	sess.Join("LEFT", "user_auth", "user_auth.user_id=u.id")
```

Comment:
Won't this cause the result to contain a row per combination of user and auth_module. Which would mean that users that have logged in with several auth modules would show up multiple times in the result?

Reply:
Hm... it seems you're right. I need to fix a query.

Reply:
@xlson I found that the only way to leave only latest auth module in SQL query is a conditional JOIN, so the resulted query will look like 
```sql
SELECT u.id, u.email, u.name, u.login, u.is_admin, u.is_disabled, u.last_seen_at, user_auth.auth_module 
FROM user as u
LEFT JOIN user_auth ON user_auth.id = (
  SELECT id from user_auth
  WHERE user_auth.user_id = u.id
  ORDER BY user_auth.created DESC
  LIMIT 1
) ...
```
Integration tests are passed, so it should work for all platforms (only not sure about MSSQL - we have no integration tests for it).

Reply:
Nice!